pdf-inspector.gemspec

```diff
@@ -16,6 +16,8 @@ Gem::Specification.new do |spec|
                 "jimmy@deefa.com"]
   spec.licenses = %w(PRAWN GPL-2.0 GPL-3.0)
   spec.add_dependency('pdf-reader', '~>1.0')
+  spec.add_development_dependency('rspec')
+  spec.add_development_dependency('simplecov')
```

Comment:
I think, we can keep RSpec since we're using it anyway. But SimpleCov is a new dep that I would skip. At least in the context of this PR.